Tasks/XamarinAndroid/Strings/resources.resjson/de-de/resources.resjson

```diff
@@ -11,6 +11,8 @@
   "loc.input.help.target": "Erstellt diese Ziele in diesem Projekt. Trennen Sie mehrere Ziele mit einem Semikolon.",
   "loc.input.label.outputDir": "Ausgabeverzeichnis",
   "loc.input.label.configuration": "Konfiguration",
+  "loc.input.label.clean": "Bereinigen",
+  "loc.input.help.clean": "Führt einen bereinigten Build (\"/t:clean\") vor dem Build aus.",
```

Comment:
Did you come up with the localized strings? You shouldn't need to do that, only update the task.json with ENU strings and rest will be generated later on.